python/ql/test/library-tests/dataflow/coverage/functional.py

```diff
@@ -0,0 +1,67 @@
+# All functions starting with "test_" should run and execute `print("OK")` exactly once.
+# This can be checked by running validTest.py.
+
+import sys
+import os
+
+sys.path.append(os.path.dirname(os.path.dirname((__file__))))
+from testlib import expects
+
+# These are defined so that we can evaluate the test code.
```

Comment:
These definitions feel like boilerplate that should be shared among tests more generally (and hence defined elsewhere -- `testlib` perhaps?), but I may be missing something about the setup.

Reply:
It is boilerplate, and actually, some of our tests do an import instead. I should change this.